src/fhir.ts

```diff
@@ -1,6 +1,5 @@
-import makeFhir from "fhir.js/src/fhir.js";
+import makeFhir from "./fhirjs";
```

Comment:
The reason I switched this import is some wild typescript magicks that i'd be happy to explain over slack or to an interested party (but would take a long time to type up). It's a hack to workaround some problems with using fhir.js@0.0.22